RetailCoder.VBE/UI/Settings/IndenterSettingsViewModel.cs

```diff
@@ -262,14 +263,72 @@ public int IndentSpaces
         }
 
         // ReSharper disable once InconsistentNaming
-        private const string _previewSampleCode = @"Sub Foo()
-    ' Do something here
-    ' xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx ' comment here
-End Sub";
+        private const string _previewSampleCode =
+@"' Example Procedure
+Sub ExampleProc()
+
+    ' SMART INDENTER
+    '@ 2016 by Rubberduck VBA.
```

Comment:
how about

    ' SMART INDENTER
    ' Original VB6 code graciously offered to Rubberduck by Stephen Bullen & Rob Bovey